demo/index.html

```diff
@@ -173,5 +181,42 @@
       </paper-tabs>
     </template>
 
+    <h3>J. Controlling content</h3>
+
+    <test-tabs></test-tabs>
+    <dom-module is="test-tabs">
+      <template>
+        <style include="paper-tabs-demo-styles"></style>
+        <style>
+          iron-pages {
+            border: 1px solid #ccc;
+            border-top: none;
+            padding: 8px;
+          }
+        </style>
+        <paper-tabs selected="{{selected}}" noink no-bar>
+          <paper-tab>ITEM ONE</paper-tab>
+          <paper-tab>ITEM TWO</paper-tab>
+          <paper-tab>ITEM THREE</paper-tab>
+        </paper-tabs>
+        <iron-pages selected="{{selected}}">
+          <div>CONTENT ONE</div>
+          <div>CONTENT TWO</div>
+          <div>CONTENT THREE</div>
+        </iron-pages>
+      </template>
+      <script>
+        Polymer({
```

Comment:
Unfortunately, this demo will break in Polyfill'd browsers. `Polymer` is not defined synchronously before this script will run, so the call to `Polymer` needs to be wrapped in:

```javascript
document.addEventListener('HTMLImportsLoaded', function() {
  // Polymer call is safe here...
});
```

Alternatively, the call can be moved to an Import - if you do this, it will not be necessary to wait for `HTMLImportsLoaded`.